aws_s3_bucket.tf

```diff
@@ -61,31 +73,47 @@ resource "aws_s3_bucket" "s3VersioningMfaFalse" {
       }
     }
   }
-  
+
   versioning {
-    enabled = false
-    mfa_delete = false
+    enabled    = true
+    mfa_delete = true
   }
 }
 
 resource "aws_s3_bucket" "allUsersReadAccess" {
   bucket = "my-tf-test-bucket"
-  acl    = "public-read"
+  acl    = "private"
+
+  versioning {
+    enabled = true
+  }
 }
 
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket should have versioning enabled**
 
**Vulnerability**
Enabling S3 versioning will enable easy recovery from both unintended user actions, like deletes and overwrites
 
**Remediation**
S3 bucket should have versioning enabled

source: accurics-